alerting/alerting-jar/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -85,97 +91,84 @@
     @Inject
     @Configurable
     @ConfigurationProperty(DISABLE_METRICS_FORWARDING)
-    private String disableMetricsForwarding;
+    private String disableMetricsForwardingProperty;
 
     @Inject
     @Configurable
     @ConfigurationProperty(DISABLE_PUBLISH_FILTERING)
-    private String disablePublishFiltering;
+    private String disablePublishFilteringProperty;
+    private boolean disablePublishFiltering;
 
     public void onMetricsServiceReady(@Observes @ServiceReady ServiceReadyEvent event) {
-        if (!Boolean.parseBoolean(disableMetricsForwarding)) {
+        if (!Boolean.parseBoolean(disableMetricsForwardingProperty)) {
             publishPeriod = getPublishPeriod();
+            disablePublishFiltering = Boolean.parseBoolean(disablePublishFilteringProperty);
             Observable<List<Metric<?>>> events;
-            if (Boolean.parseBoolean(disablePublishFiltering)) {
-                events = event.getInsertedData()
-                        .buffer(publishPeriod, TimeUnit.MILLISECONDS, 100)
-                        .filter(list -> !list.isEmpty());
-            } else {
-                events = event.getInsertedData()
-                        .filter(m -> m.getType() != MetricType.STRING)
-                        .buffer(publishPeriod, TimeUnit.MILLISECONDS, 100)
-                        .filter(list -> !list.isEmpty());
-            }
+            events = event.getInsertedData()
+                    .buffer(publishPeriod, TimeUnit.MILLISECONDS, 100)
```

Comment:
If this PR was motivated for a performance test scenario, the buffer should have more control from configuration side (something where we can pass some config to see how buffering might affect to different pressure scenarios IMHO).